packages/plugin-devtools/src/components/InfoComponent.ts

```diff
@@ -34,6 +35,31 @@ export class InfoComponent extends OwlUIComponent<{}> {
     className(vNode: VNode): string {
         return vNode.constructor.name;
     }
+    /**
+     * Return an object representing the given VNode's public properties as
+     * alphabetically sorted pairs of key and value strings.
+     *
+     * @param vNode
+     */
+    nodeCustomProperties(vNode: VNode): { [key: string]: string }[] {
+        return Object.keys(vNode)
+            .filter(key => !key.startsWith('_') && !vNodeKeys.includes(key))
+            .sort()
+            .map(key => {
+                let value = '' + vNode[key];
+                if (typeof vNode[key] === 'object') {
+                    if (!vNode[key] || !Object.keys(vNode[key]).length) {
+                        value = '{}';
+                    } else if (vNode[key].toString === {}.toString) {
+                        value = this._objectRepr(vNode[key]);
+                    }
+                }
+                if (typeof vNode[key] === 'string') {
```

Comment:
if => else if

Reply:
True semantically but in practice it can't possibly return true on both conditions. For micro-performance management though you are right. Applied.